### PR TITLE
added 4 practice

### PR DESCRIPTION
**Pull request by IrenZh**

### CODE REVIEW

File: practice_4/task_3.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+let age = 14;
+
+function checkAge(age) {
+    try {
+    if (!age){
+        throw new Error ('The field is empty!  Please enter your age');
```
Comment (VolodymyrLykhachov) [inline practice_4/task_3.js]: You need to add not only message of error but it name too
throw{
        name: 'Error name',
        message: 'Error message',
      };

File: practice_4/task_5.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+function showUser(id){
```
Comment (VolodymyrLykhachov) [inline practice_4/task_5.js:8]: There should be class MonthException. You need to implement it

File: practice_4/task_3.js (hunk 1 above)
Comment (IrenZh) [reply practice_4/task_3.js]: done

File: practice_4/task_5.js (hunk 2 above)
Comment (IrenZh) [reply practice_4/task_5.js:8]: If I understand correct, you write about IdException class
So I implemented it.

**Review by VolodymyrLykhachov — APPROVED** [review]

**Pull request merged**